src/pages/deriv-go/components/_new-app.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react'
+import { graphql, useStaticQuery } from 'gatsby'
+import styled from 'styled-components'
+import { Localize } from 'components/localization'
+import { Header, QueryImage } from 'components/elements'
+import { Flex } from 'components/containers'
+import device from 'themes/device'
+
+const Section = styled.div`
+    display: flex;
+    flex-direction: row;
+    width: 100%;
+    height: 200px;
+    background-color: #f5f7fa;
+
+    @media ${device.tablet} {
+        padding: 0 16px 40px 16px;
+    }
+`
+
+const StyledHeader = styled(Header)`
+    line-height: 1.25;
+    font-size: 3rem !important;
+`
+
+const ImageWrapper = styled(Flex)`
+    display: flex;
+`
+
+const Left = styled.div`
+    margin-right: 1.6rem;
+    margin-top: 40px;
+    margin-bottom: 40px;
+`
+
+const Right = styled.div`
+    justify-content: flex-end;
+    margin-top: 52px;
+    margin-bottom: 52px;
+`
+
+const query = graphql`
+    query {
+        dgo_logo: file(relativePath: { eq: "derivgo/dgo_logo.png" }) {
+            ...fadeIn
+        }
+        qr_code: file(relativePath: { eq: "derivgo/qr_code.png" }) {
+            ...fadeIn
+        }
+    }
+`
```

Comment:
These image files need to be included as well

Reply:
okay, I'll add the images